src/engine/GroupBy.cpp

```diff
@@ -1034,14 +1042,31 @@ concept SupportedAggregates =
     ad_utility::SameAsAnyTypeIn<A, GroupBy::AggregationDataVectors>;
 
 // _____________________________________________________________________________
-std::vector<size_t> GroupBy::HashMapAggregationData::getHashEntries(
-    std::span<const Id> ids) {
-  std::vector<size_t> hashEntries;
-  hashEntries.reserve(ids.size());
+template <size_t GROUPED_COLUMNS>
+std::vector<size_t>
+GroupBy::HashMapAggregationData<GROUPED_COLUMNS>::getHashEntries(
+    const T<std::span<const Id>> ids) {
```

Comment:
It is not really `ids` but `groupByCols` or something (each entry is a subset of a column).